gruntfile.js

```diff
@@ -31,7 +31,7 @@ module.exports = function(grunt){
         extensions: "js",
         specNameMatcher: "[Ss][Pp][Ee][Cc][Ss]",
         jUnit: {
-          report: true,
+          report: false,
```

Comment:
I have also got this problem jasmine-contrib/grunt-jasmine-node#52 and I found you already commented there. So is this a temporary solution?